README.md

```diff
@@ -1,16 +1,19 @@
 # What is Multipass?
 
-Multipass is a lightweight VM manager for Linux, Windows and macOS. It's designed for developers who want to spin up a fresh Ubuntu environment with a single command. It uses KVM on Linux, Hyper-V on Windows and QEMU on macOS to run virtual machines with minimal overhead. It can also use VirtualBox on Windows and macOS. Multipass will fetch Ubuntu images for you and keep them up to date.
+Multipass is a lightweight VM manager for Linux, Windows and macOS. It's designed for developers who want to spin up a
```

Comment:
Are these changes an artifact of the new line width linting?

Reply:
Not really, clang-format did not affect these files. This was me explicitly asking CLion to reformat and then manually tweaking here and there.